### ISSUE TITLE
Change shortcode builder default format

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/boardscribe#46]
This format is better for screen readers, so I think it should be preset that way in the shortcode builder:

-  `l, F j, Y` for Held Date
- `F Y` for Not Held Date
